e2e/global.setup.ts

```diff
@@ -0,0 +1,90 @@
+setup('login and create account', async ({ page }) => {
+  const { SPLUNK_USER: user, SPLUNK_PASSWORD: password, CREATE_ACCOUNT: createAccount } = process.env;
+
+  // Login
+  await page.goto('/');
+  await page.screenshot({ path: 'playwright-screenshots/page-login.png', fullPage: true });
+
+  console.log("Fill in login form");
+  await page.getByPlaceholder('Username').fill(user || 'SPLUNK_USER env is empty');
+  await page.getByPlaceholder('Password',  { exact: true }).fill(password || 'SPLUNK_PASSWORD env is empty');
+  await page.getByRole('button', { name: 'Sign In' }).click();
+
+  console.log("Wait for homepage to load");
+  await expect(page).toHaveTitle(/Home/);
+
+  await page.screenshot({ path: 'playwright-screenshots/page-after-login-before-pop-up.png', fullPage: true });
+
+  console.log("Store state to be able to reuse login");
+  await page.context().storageState({ path: STORAGE_STATE });
+
+  await page.waitForTimeout(2000);
+  // Confirm some pop-up
+  const locGotIt = page.getByTestId('instrumentation-opt-in-modal').locator('[data-test="button"]');
+  const countGotIt = await locGotIt.count()
+  console.log("Check for popup: ", countGotIt);
+  if (countGotIt > 0) {
+    console.log("Close the popup");
+    await locGotIt.click();
+  }
+
+  await page.screenshot({ path: 'playwright-screenshots/page-after-login-after-pop-up.png', fullPage: true });
+
+  await goToDataSet(page);
+
+  await page.screenshot({ path: 'playwright-screenshots/page-home.png', fullPage: true });
+
+  console.log("Go to configuration page");
+  await page.locator('[title="Configuration"]').click();
+
+  // Check that we are on the configuration page
+  await expect(page).toHaveTitle(/Configuration/);
+  await page.screenshot({ path: 'playwright-screenshots/page-configuration.png', fullPage: true });
+
+  // wait for table to appear
+  await page.getByText(/Account name/).click()
+
+  const accountCount = await page.getByRole("main").getByRole("row").getByText(/QaTr/).count();
+  console.log("Number of accounts: ", accountCount);
+  if (accountCount == 0) {
+    // Open dialog
+    console.log("Open dialog for adding an account");
+    const locAddDialog = page.locator('#accountTab').getByRole('button', { name: 'Add' });
+    await locAddDialog.click();
+
+    // Setup locators
+    const locAccount = page.locator('div').filter({ hasText: /^Account nameEnter a unique name for this account\.$/ }).locator('[data-test="textbox"]');
+    const locUrl = page.locator('div').filter({ hasText: /^URLEnter DataSet URL\.$/ }).locator('[data-test="textbox"]');
+    const locReadKey = page.locator('[data-test="body"] form div').filter({ hasText: 'DataSet Log Read Access KeyRequired to enable inputs and SPL comand. Include tra' }).locator('[data-test="textbox"]');
+    const locWriteKey = page.locator('[data-test="body"] form div').filter({ hasText: 'DataSet Log Write Access KeyRequired to enable alert action. Include trailing hy' }).locator('[data-test="textbox"]');
```

Comment:
Using general selectors like `form` or `div` is the least optimal way, because they are prone to change which breaks the tests. Using `#accountTab` (selector by id, should be unique) would be better. I understand in terms of FE we have to work with what we have, so there might not be a better way how to do this.